docs/rootfs-and-kernel-setup.md

```diff
@@ -185,3 +185,45 @@ adjust the script(s) to suit your use case.
 
 You should now have a rootfs image (`ubuntu-22.04.ext4`), that you can boot with
 Firecracker.
+
+## Creating FreeBSD rootfs and kernel Images
+
+Here's a quick step-by-step guide to building a FreeBSD rootfs and kernel that
+Firecracker can boot:
+
+1. Boot a FreeBSD system.  In EC2, the
+   [FreeBSD 13 Marketplace image](https://aws.amazon.com/marketplace/pp/prodview-ukzmy5dzc6nbq)
+   is a good option; you can also use weekly snapshot AMIs published by the
+   FreeBSD project.  (Firecracker support is in FreeBSD 14 and later, so you'll
+   need FreeBSD 13 or later to build it.)
+
+   The build will require about 50 GB of disk space, so size the disk
+   appropriately.
+
+1. Log in to the FreeBSD system and become root.  If using EC2, you'll want to
+   ssh in as `ec2-user` with your chosen SSH key and then `su` to become root.
+
+1. Install git and check out the FreeBSD src tree:
+
+   ```sh
+   pkg install -y git
+   git clone https://git.freebsd.org/src.git /usr/src
+   ```
+
+   At present (July 2023) Firecracker support is only present in the `main`
+   branch.
```

Comment:
Not very present anymore.

Reply:
I think here we can say to clone 14.0 version since it is one with FC support

Reply:
changed